packages/client/lib/commands/SORT_RO.ts

```diff
@@ -0,0 +1,52 @@
+export const FIRST_KEY_INDEX = 1;
+
+export const IS_READ_ONLY = true;
+
+export interface SortReadOnlyOptions {
+    BY?: string;
+    LIMIT?: {
+        offset: number;
+        count: number;
+    },
+    GET?: string | Array<string>;
+    DIRECTION?: 'ASC' | 'DESC';
+    ALPHA?: true;
+}
+
+export function transformArguments(key: string, options?: SortReadOnlyOptions): Array<string> {
+    const args = ['SORT_RO', key];
+
+    pushSortReadOnlyArgs(args, options);
+
+    return args;
+}
+
+export function pushSortReadOnlyArgs(args: Array<string>, options?: SortReadOnlyOptions) {
```

Comment:
@leibale Should I move it or leave it here?

Reply:
@Avital-Fine move it to `generic-transformers.ts` please